docs/pages/versions/unversioned/sdk/camera.mdx

```diff
@@ -82,12 +82,12 @@ Learn how to configure the native projects in the [installation instructions in
 > **warning** Only one Camera preview can be active at any given time. If you have multiple screens in your app, you should unmount `Camera` components whenever a screen is unfocused.
 
 ```jsx
```

Comment:
Let's also change the snippet language to `tsx`.

Reply:
Thank you for your CR! Have done them.